packages/python-modules/cryptography36/default.nix

```diff
@@ -0,0 +1,76 @@
+{ pkgs }:
+
+let
+  cryptography-vectors = pkgs.callPackage ./vectors.nix { pkgs=pkgs;};
+in
+with pkgs;
+python3Packages.buildPythonPackage rec {
```

Comment:
Is this based on some original package in the `nixpkgs` repo? When you fork a package, please include a link to the original, so it's a bit easier to study the code and the nature of the changes that we introduce.

Reply:
Added comment with link